runtime/compiler/runtime/SymbolValidationManager.cpp

```diff
@@ -68,6 +68,8 @@ TR::SymbolValidationManager::SymbolValidationManager(TR::Region &region, TR_Reso
      _idToSymbolsMap((IdToSymbolComparator()), _region),
      _seenSymbolsSet((SeenSymbolsComparator()), _region)
    {
+   assertionsAreFatal(); // Acknowledge the env var whether or not assertions fail
```

Comment:
This call exists only to set the static bool right? I'm only clarifying because `assertionsAreFatal()` returns a bool, but the return value is ignored here.

Reply:
Yes, we don't need the result here. We don't even really need to set the static boolean yet, since its value doesn't matter until an assertion fails. So this is only here to acknowledge the environment variable. That way, if a user is looking for `JIT: env var TR_svmAssertionsAreFatal is set...`, they'll see it even if no assertions fail, which is the circumstance where that information is useful. (If the VM is crashing with an SVM assertion failure, then of course that SVM assertion was fatal.)